core/invoker/src/main/scala/org/apache/openwhisk/core/containerpool/ContainerPool.scala

```diff
@@ -32,10 +36,14 @@ sealed trait WorkerState
 case object Busy extends WorkerState
 case object Free extends WorkerState
 
+case class ColdStartKey(kind: String, memory: Long, date: String)
```

Comment:
What is date used for? 
Memory should be ByteSize for consistency elsewhere.

Reply:
Updated accordingly